docs/enhancements/backup-restore-support.md

```diff
@@ -0,0 +1,152 @@
+I.e. the assisted-service should be resilient in such scenarios which, for this context and effort, means that restored/moved spoke clusters should keep the same state and behave the same on the new hub cluster. 
+
+## Motivation
+
+Provide resiliency in the assisted-service for safe backup/restore flows, allowing spoke clusters to be used without any restriction after DR scenarios or moving between hubs.
+
+### Goals
+
+Spoke clusters (with/without BMHs) should have a consistent state and function properly, including the ability to add/remove nodes, after these scenarios: 
+* Spoke clusters restored on a passive hub with ACM/OADP functionality.
+* Spoke clusters moved between hubs using Kube-API.
+
+### Non-Goals
+
+* Improve import day2 cluster solution by getting full host data (e.g. using an agent on existing nodes).
+
+## Proposal
+
+### User Stories
+
+#### Story 1
+
+As an Infrastructure Admin, I want to backup an installed spoke cluster from an active hub, including the associated BMHs (e.g. by using ACM/Kube-API/GitOps/etc).
+Then, I want to safely restore the cluster on a passive hub, including the BMHs that should keep the same state. I.e. the restored cluster should behave exactly as in the active hub with no restrictions (support day2 operations, etc).
+Note: if unbound hosts are included, they should still be automatically reprovisioned using a new discovery ISO.
+
+#### Story 2
+
+As an Infrastructure Admin, I want to backup an installed spoke cluster that doesn't have associated BMHs. Then, I want to safely restore the cluster on a passive hub.
+Note: if unbound hosts are included, they should be manually booted using a new discovery ISO. 
+
+#### Story 3
+
+As an Infrastructure Admin, I want to restore a spoke cluster in non-installed status (draft/in-progress/etc). The cluster on target hub should, ideally, keep the same status. Or, at least, fail gracefully and let the user retry the installation.
+Unbound hosts should be treated as previous stories (depends on BMHs).
+
+#### Story 4
+
+As an Infrastructure Admin, I want to safely move installed spoke clusters between hubs.
+Concerns are similar to previous stories.
+
+### Implementation Details/Notes/Constraints
+
+#### Solution for BMHs reconcile (due to a missing status)
+
+According to BMO documentation, the suggested approach for moving BareMetalHost from one cluster to another is using the `baremetalhost.metal3.io/status` [annotation](https://github.com/metal3-io/metal3-docs/blob/main/docs/user-guide/src/bmo/status_annotation.md).
+I.e. Extract the current status of the BMH, store it in an annotation, create the BMH on the new hub:
+```
+# Save the status in json format to a file
+kubectl get bmh <name-of-bmh> -o jsonpath="{.status}" > status.json
+# Save the BMH and apply the status annotation to the saved BMH.
+kubectl -n metal3 annotate bmh <name-of-bmh> \
+  baremetalhost.metal3.io/status="$(cat status.json)" \
+  --dry-run=client -o yaml > bmh.yaml
```

Comment:
I assume  you shouldn't include this `--dry-run` bit when doing this in a real environment, right?

Reply:
So according to the doc, the idea is to first just create the bmh.yaml file. Then, apply it to the target hub.
I'll add a note to clarify that.

Reply:
Ah I just missed this was creating a file then applying it. Nevermind, you can ignore this.